common/src/stack/command/stack/commands/add/host/interface/__init__.py

```diff
@@ -161,7 +161,19 @@ def run(self, params, args):
 			# The auto ip feature requires a network parameter
 			if subnet_id is None:
 				raise ParamRequired(self, 'network')
-		
+
+		# Unset other default interface if the added one is the default interface
+		if default:
+			self.db.execute("""
+				update networks
+				set main=0
+				where node=(select id from nodes where name=%s)
+				and main=1
+				""",
+				(host)
+				)
+
+
```

Comment:
I think you are missing a comma after `host` in the args you are passing along with the SQL. A single item in parentheses is the same as just the item without them. To make it a tuple, it always needs a comma, like so: `(host,)`

I think the `and main=1` part is redundant, since the column will only be 0 or 1.

Also, IMO the SQL doesn't need to be broken into so many lines. This reads better to my eyes:
```
			self.db.execute("""
				update networks set main=0
				where node=(select id from nodes where name=%s)
			""", (host,)
			)
```